packages/terra-i18n/docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Terra I18n
+
+The terra-i18n component provides the internationalization to the React component. Terra supports the following locales: 'de' 'es' 'en' 'en-US' 'en-GB', 'fi-FI', 'fr', 'pt'. All locales related fils are loading on demand.
```

Comment:
Fixed the typo: https://github.com/cerner/terra-ui/pull/141/commits/69beebb1e31ea04107e911235d32b2af146216e5